Gemfile

```diff
@@ -22,6 +22,7 @@ group :test do
   gem 'rspec', '>= 3.2'
   gem 'simplecov', '>= 0.11.2'
   gem 'yarjuf', '>= 2.0'
+  gem "rubocop", :platforms => [:ruby]
```

Comment:
Maybe add a comment about Rubocop being A-OK if we transition to Jruby 9000 but for now it's MRI only?